src/strategies/result.test.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash/fp'
```

Comment:
this file name should be consistent with the thing its testing - `results.test.js` instead of `result.test.js`

Reply:
good catch.